### PR TITLE
Review functests and re-enable them

### PR DESCRIPTION
**Pull request by jschmid1**
These are the functests that are currently passing.
We will extend this list when more things are working again

-----------------

**Checklist:**
- [ ] Ran integration tests successfully (trigger with "@susebot run teuthology" in a GitHub comment; see the [wiki](https://github.com/SUSE/DeepSea/wiki/Testing) for more information)

### CODE REVIEW

**Comment by smithfarm** [comment]
@susebot run teuthology deepsea/tier3/functional

**Comment by jschmid1** [comment]
@susebot run teuthology deepsea/tier3/functional

**Comment by smithfarm** [comment]
@jschmid1 This time the test failed because tuned daemon was not running. This is because tuned is disabled by default in all tests - for the functional tests I could enable it, or would you rather disable the tuned functional test?

**Comment by smithfarm** [comment]
(tuned was disabled because it was causing no end of instability in the CI)

**Comment by jschmid1** [comment]
I'd avocate for enabling it for the functests to see if it is stable in the meanwhile

**Comment by smithfarm** [comment]
@susebot run teuthology deepsea/tier3/functional

**Comment by smithfarm** [comment]
@jschmid1 Fails in "Shutdown ganesha". There is no ganesha role - should I add it?

**Comment by smithfarm** [comment]
(keeping in mind this is a single-node cluster, I hesitate to add ganesha for fear of introducing instability)

**Comment by jschmid1** [comment]
> @jschmid1 Fails in "Shutdown ganesha". There is no ganesha role - should I add it?

What about moving a couple of tests to (like migrate and terminate) to a 3node setup?

**Comment by jschmid1** [comment]
I may split the `termniate` in two pieces. `core` and `services` where the latter will only executed in a n>1 environment?

**Comment by smithfarm** [comment]
Yes, my thoughts exactly. The tests are currently in `srv/salt/ceph/functests/1node`. We can also have, e.g., `srv/salt/ceph/functests/2nodes` and/or `srv/salt/ceph/functests/3nodes`.

**Comment by smithfarm** [comment]
@susebot run teuthology deepsea/tier3/functional

**Comment by smithfarm** [comment]
@susebot run teuthology deepsea/tier3/functional

**Comment by smithfarm** [comment]
In the end I just enabled the `ganesha` role on the cluster node (along with the `prometheus` and `grafana` roles, so the CI is ready for functests exercising those roles).

File: srv/salt/ceph/functests/1node/terminate/ganesha.sls [hunk 1]
```diff
@@ -7,7 +6,7 @@ Shutdown ganesha on {{ node }}:
     - sls: ceph.terminate.ganesha
     - failhard: True
 
-Check process:
+Check process for for nfs-ganesha:
```
Comment (smithfarm) [inline srv/salt/ceph/functests/1node/terminate/ganesha.sls]: s/for for/for/

**Comment by jschmid1** [comment]
great, thanks @smithfarm 
merging..

**Pull request merged**